### PR TITLE
@dzucconi => Add markdown

### PR DESCRIPTION
**Pull request by mzikherman**
Now you can do:

```
{
  partner_show(id: "new-museum-1-2015-triennial-surround-audience") {
    press_release(markdown: true)
  }
}

// =>
{
  "data": {
    "partner_show": {
      "press_release": "<p>My very nice HTML</p>"
    }
  }
}
```

### CODE REVIEW

**Comment by dzucconi** [comment]
I find that argument confusing. Like one would expect `markdown: true` to return Markdown? But what if there is more than one format, now we have conflicting arguments.

Let's do it by setting up an `EnumType` of valid formats and then do `field(format: "html")` or `field(format: "md")`

**Comment by mzikherman** [comment]
I like that, k.

**Comment by mzikherman** [comment]
Updated to support one format (for now!).

So you do:

```
press_release(format: markdown)
```

File: test/schema/partner_show.js [hunk 1]
```diff
@@ -46,4 +47,26 @@ describe('PartnerShow type', () => {
...
+        PartnerShow.__get__('gravity').args[0][0]
+          .should.equal('show/new-museum-1-2015-triennial-surround-audience');
+
+        data.should.eql({
+          partner_show: {
+            press_release: '<p><strong>foo</strong> <em>bar</em></p>\n'
+          }
+        })
+      });
+  });
```
Comment (dzucconi) [inline test/schema/partner_show.js]: Very nice

**Pull request merged**